Assets/Scripts/Control&Input/AIMovement.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Collections;
+using UnityEngine;
+
+public class AIMovement : PlayerMovement
+{
+    void Start()
+    {
+        body = GetComponent<Rigidbody>();
+        hitbox = GetComponent<BoxCollider>();
+        strafeForce = strafeForceGrounded;
+    }
+    private void Update()
+    {
+        UpdateAnimatorParameters();
+    }
+
+    private void FixedUpdate()
+    {
+    }
+
+    public void Jump()
+    {
+        if (StateIsAir)
+            return;
+        body.AddForce(Vector3.up * jumpForce, ForceMode.VelocityChange);
```

Comment:
Isn't this replaced by an animation?

Reply:
This is an unused function intended for #459, as this script needs to take over control onDestination reached for player targets in order to emulate strafeing, jumping and distance keeping. I am postponing actually implementing this for the time being

Reply:
Added a TODO in the code so it isn't left hanging